searx/templates/simple/result_templates/packages.html

```diff
@@ -58,6 +58,14 @@
         {%- if result.source_code_url -%}
           <a href="{{ result.source_code_url }}" target="_blank">{{ _('Source code') }}</a>
         {%- endif -%}
+        {%- if result.links %}
+          {%- for name, link in result.links.items() -%}
+            {% if not loop.first or result.homepage or result.source_code_url %} | {% endif %}
+            <a href="{{ link }}" target="_blank">
+              {{- _(name) -}}
+            </a>
+          {%- endfor -%}
```

Comment:
I added some `{%-` & `-%}` to get rid of not needed whitespaces in the output.

Reply:
Should this be normalised: upper or lower?

Reply:
Good question .. I hesitate to normalize in the templates .. upper-/lowercase depends on the l10n .. we have languages that do not follow the rules of the alphabet (chinese, hebraic, arabic ..) .. in my opinion, a template should be neutral with regard to i18n.

Normalization should therefore better be done in the engine ... if hex does not return a normalized value (if some results contain “Author” and others “author”) we would have to normalize this in the hex engine ... I pushed https://github.com/searxng/searxng/pull/3464/commits/5269dbdd053cc57c5cb40e4a606a7c2351730687 on top of this branch.

Is it OK to merge this commit with your PR here?

Reply:
it's ok.